config/locales/physical_instantiation.de.yml

```diff
@@ -0,0 +1,8 @@
+de:
+  hyrax:
+    icons:
+      physical_instantiation:     'fa fa-file-text-o'
```

Comment:
Do we know what `fa fa-file-text-o` is supposed to be? Is it an HTML ID or class value? And if so, why is it in a localization file?

Reply:
@afred i believe `fa fa-file-text-o` is css class for file icon.